code/modules/projectiles/projectile/energy.dm

```diff
@@ -82,19 +85,21 @@
 	nodamage = 0
 	agony = 30
 	stutter = 10
+	recoil = 3
 
 
 /obj/item/projectile/energy/bolt/large
 	name = "largebolt"
 	damage_types = list(BURN = 25)
+	recoil = 5
 
 /obj/item/projectile/energy/neurotoxin
 	name = "neuro"
 	icon_state = "neurotoxin"
 	damage_types = list(TOX = 5)
 	weaken = 5
 
-/obj/item/projectile/energy/plasma
+/obj/item/projectile/energy/plasma // What?
```

Comment:
?

Reply:
It's an unused bay leftover gun that's called a plasma gun despite having nothing to do with plasma guns.